README.md

```diff
@@ -24,3 +24,26 @@ Now, edit your project's `/package.json` file to contain the following:
   "node": ">=12.16.1"
 }
 ```
+
+## Homey App config
+
+This package also contains a ESLint config for Homey Apps, you can extend it like this:
+
+```javascript
+{
+  "extends": "athom/homey-app"
+}
+```
+
+This config includes some rules that depend on TypeScript typechecking so you need to make sure to have a `tsconfig.json` file in the root of your Homey App.
```

Comment:
Are there any requirements as to the contents of `tsconfig.json` the developer should know about?

Reply:
Mhh not really although it should be usable by typescript. Will improve these instructions when I add a command to the Homey CLI to automate this.